Proiect/src/MenuRestaurant.java

```diff
@@ -0,0 +1,32 @@
+import java.util.HashMap;
+import java.util.Map;
+
+public class MenuRestaurant
+{
+    protected Map<String, Integer> restaurant_menu;
+
+    public  MenuRestaurant()
+    {
+        restaurant_menu = new HashMap<String, Integer>();
+    }
+
+    public Map<String, Integer> getRestaurant_menu()

```

Comment:
getRestaurantName and the same for all of the methods and fields you have in the projects - camelCase rule